minecraft_servers/lalura/manifest.json

```diff
@@ -0,0 +1,33 @@
+{
+    "server_name": "lalura",
+    "nice_name": "Lalura",
+    "direct_ip": "lalura.net",
+    "server_wildcards": [
+      "%.lalura.net"
+    ],
+    "supported_languages": [
+       "de"
+    ],
+    "social": {
+      "web": "https://lalura.net",
```

Comment:
We only accept working websites. Please fix or remove (you can still add the page later).
![Bildschirmfoto 2023-07-24 um 08 15 26](https://github.com/LabyMod/server-media/assets/45363287/98e70c58-491d-49e2-80d4-99d1bc2a82bc)